source/Octopus.Cli/Commands/InstallAutoCompleteCommand.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Markdig.Extensions.ListExtras;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Util;
+using Serilog.Events;
+
+namespace Octopus.Cli.Commands
+{
+    public enum SupportedShell
+    {
+        Pwsh,
+        Zsh,
+        Bash,
+    }
+    
+    [Command(name: "install-autocomplete", Description = "Install a shell auto-complete script into your shell profile, if they aren't already there. Supports Powershell (pwsh), Z Shell (zsh), Bourne Again Shell (bash) & Friendly Interactive Shell (fish).")]
+    public class InstallAutoCompleteCommand : CommandBase, ICommand
+    {
+        private readonly IOctopusFileSystem fileSystem;
+
+        public InstallAutoCompleteCommand(ICommandOutputProvider commandOutputProvider, IOctopusFileSystem fileSystem) : base(commandOutputProvider)
+        {
+            this.fileSystem = fileSystem;
+
+            var options = Options.For("Install AutoComplete");
+            options.Add("shell=",
```

Comment:
This feels mandatory... It defaulted to pwsh for me on pwsh, but i'm pretty sure that's by chance.
it should go :boom: if this arg isn't provided

Reply:
Good spot will fix